### PR TITLE
Refactor `SkipLink.tsx` component to use ChakraUI

### PR DESCRIPTION
**Pull request by MahendraBishnoi29**
## Description
Refactored `SkipLink.tsx` component with Chakra UI

## Related Issue
#6374

### CODE REVIEW

File: src/components/SkipLink.tsx [hunk 1]
```diff
@@ -1,43 +1,31 @@
...
   return (
-    <Div>
-      <Anchor href={hrefId}>
+    <Box bg="primary">
+      <Link
+        href={hrefId}
+        lineHeight="2rem"
+        position="absolute"
+        top="-3rem"
+        ml=".5rem"
```
Comment (pettinarip) [inline src/components/SkipLink.tsx:17]:
```suggestion
        ml="2"
```

File: src/components/SkipLink.tsx [hunk 2]
```diff
@@ -1,43 +1,31 @@
...
 }
 
 export const SkipLink: React.FC<IProps> = ({ hrefId }) => {
   return (
-    <Div>
-      <Anchor href={hrefId}>
+    <Box bg="primary">
+      <Link
+        href={hrefId}
+        lineHeight="2rem"
```
Comment (pettinarip) [inline src/components/SkipLink.tsx:14]: https://chakra-ui.com/docs/styled-system/theme#typography
```suggestion
        lineHeight="taller"
```

File: src/components/SkipLink.tsx [hunk 3]
```diff
@@ -1,43 +1,31 @@
...
-  &:focus {
-    position: static;
-  }
-`
-
-const DivAnchor = styled.div`
-  height: 80px;
-  margin-top: -80px;
-`
+import { Box, Link } from "@chakra-ui/react"
```
Comment (pettinarip) [inline src/components/SkipLink.tsx:3]: Let use the custom `Link` component from `src/components/Link`

File: src/components/SkipLink.tsx [hunk 4]
```diff
@@ -1,43 +1,31 @@
...
 export const SkipLink: React.FC<IProps> = ({ hrefId }) => {
   return (
-    <Div>
-      <Anchor href={hrefId}>
+    <Box bg="primary">
+      <Link
+        href={hrefId}
+        lineHeight="2rem"
+        position="absolute"
+        top="-3rem"
```
Comment (pettinarip) [inline src/components/SkipLink.tsx:16]:
```suggestion
        top="-12"
```

**Review by pettinarip — CHANGES_REQUESTED** [review]
Looking good @MahendraBishnoi29 I've just added some minor requests.

**Comment by MahendraBishnoi29** [comment]
@pettinarip is everything ok now?

**Review by pettinarip — APPROVED** [review]
lgtm! thanks @MahendraBishnoi29 💪🏼

**Pull request merged**